pvlib/modelchain.py

```diff
@@ -260,6 +263,27 @@ def get_orientation(strategy, **kwargs):
     return surface_tilt, surface_azimuth
 
 
+@dataclass
+class ModelChainResult:
```

Comment:
Good idea, but somewhat more difficult that it appears. Maybe someone more familiar with the RTD sphinx theme could help me with this:

```
Theme error:
An error happened in rendering the page generated/pvlib.modelchain.ModelChainResult.__init__.
Reason: OSError('could not get source code')
```

I think this is because the RTD theme includes listings of methods, but `ModelChainResult` doesn't have an `__init__` method. Is there any way of disabling the listing for a particular class?

Reply:
This pops up in the `get_linenos()` function in `conf.py`. Could probably work around it by adding
```python
except OSError:
    return None, None
```

Reply:
Yep, good solution.